chukwuemekachm/utility_manager_web
# Prof: User views success message on auth success

#### What does this PR 
- adds a navigation state that controls navigation from one page
to the next
- creates a PageWrapper component that retrieves data from the
connected-react-router to the navigation state for all pages in the app.
- adds sagas for navigation state
- adds reducers for navigation state

#### Any background context?
- When a page loads, the info stored in `connected-react-router` is retrieved and stored in the `navigation` state. Any page that wants to use data from previous pages would have to connect to `navigation` state

#### Manual Testing Steps
- Sign up a user via the index route
- On success, you should be redirected to the Auth success page and `userData` stored in the `navigation` state
- When you refresh the page and check the state, you reload the state you should still see `userData` in navigation state

#### Screenshots:
<img width="1440" alt="Screenshot 2019-11-29 at 9 24 00 AM" src="https://user-images.githubusercontent.com/38565349/69855337-98b56400-128b-11ea-82b0-abcfdfe6f48c.png">
<img width="1440" alt="Screenshot 2019-11-29 at 9 37 46 AM" src="https://user-images.githubusercontent.com/38565349/69855506-f47fed00-128b-11ea-8dfc-04b73905d6a2.png">
<img width="1440" alt="Screenshot 2019-11-29 at 9 37 18 AM" src="https://user-images.githubusercontent.com/38565349/69855508-f47fed00-128b-11ea-8c8c-90e112c5758e.png">
<img width="1436" alt="Screenshot 2019-11-29 at 9 36 04 AM" src="https://user-images.githubusercontent.com/38565349/69855509-f47fed00-128b-11ea-90f7-16b2723ef68e.png">


#### Relevant Trello Tickets
- [Auth Success](https://trello.com/c/jawHAM1o/37-implement-authentication-success-feedback)
